psc-publish/Main.hs

```diff
@@ -21,8 +26,13 @@ main = execParser opts >> publish
   version :: Parser (a -> a)
   version = abortOption (InfoMsg (showVersion Paths.version)) $ long "version" <> help "Show the version number" <> hidden
 
-publish :: IO ()
-publish = do
-  pkg <- preparePackage
-  BL.putStrLn (A.encode pkg)
-
+publish :: Bool -> IO ()
```

Comment:
This all looks good, but do you think it's worth introducing an Options type now, so that we don't have to break the API again later?